src/WebView.ios.tsx

```diff
@@ -37,7 +37,7 @@ const processDecelerationRate = (
   decelerationRate: DecelerationRateConstant | number | undefined,
 ) => {
   let newDecelerationRate = decelerationRate;
-  if (newDecelerationRate === 'normal') {
+  if (newDecelerationRate === 'normal' || newDecelerationRate === undefined) {
```

Comment:
Is this really necessary? Looks like the native iOS code already handles this: https://github.com/react-native-webview/react-native-webview/blob/master/apple/RNCWebViewManager.m#L163

Reply:
It might be a better way to do it. But without it - the package defaults to fast and gets the wrong scroll :( Thank you for looking at it so fast! :)

Reply:
> the package defaults to fast and gets the wrong scroll

You've verified this? Do you know how that's happening? Is the iOS code not working?

Reply:
Yes, I have ran the code in react native - iOS simulator and iOS 13 Pro device. 
Native WebView defaults to "normal", but this package defaults to "fast"

Reply:
Ah, I see. The code I linked doesn't get called at all by default, and the `decelerationRate` that it is setting is a custom property of `RNCWebView`, which is not initialized to a default value (and thus being treated similar to "fast"). I think it might be better to initialize that variable to the "normal" constant (i.e. in `initWithFrame`, where other such properties are initialized to default values). Regardless of whether the default should be "fast" or "normal," using an uninitialized float just seems like a bug.

That does mean that this is more of a breaking change though... while "normal" does seem more correct and what might be expected by new users, it's possible that existing uses are relying on the "fast" behavior...

Reply:
Good point. I'm setting in upon `initWithFrame` now and its working as expected on Simulator(iPhone 14 iOS 16.2) 👍

And yes.. it could be that someone rely on the wrong default behaviour, but that anti pattern kinda deserve a breaking change. I bet way more people will be happy that their apps scroll work as native apps now.